changelog/7263.removal.md

```diff
@@ -0,0 +1,13 @@
+The [Pika Event Broker](event-brokers.mdx#pika-event-broker) no longer supports
+the environment variables `RABBITMQ_SSL_CA_FILE` and `RABBITMQ_SSL_KEY_PASSWORD`.
+You can alternatively specify them in RabbitMQ connection url as described in the 
+[RabbitMQ documentation](https://www.rabbitmq.com/uri-query-parameters.html).
+
+```yaml-rasa title="endpoints.yml
+event_broker:
+ type: pika
+ url: "amqps://user:password@host?cacertfile=path_to_ca_cert&password=private_key_password"
```

Comment:
I'm not sure about the `password` part. It's not officially documented 😬

Reply:
maybe makes sense to remove it then

Reply:
by the way, why does the password specified twice in this string?

Reply:
The second one is hopefully the one for the private key 🤔

Reply:
removed the `password` stuff and made it clear that it's not longer supported.